wandb/sdk/wandb_settings.py

```diff
@@ -751,7 +763,12 @@ def __init__(self, **kwargs: Any) -> None:
 
         if os.environ.get(wandb.env.DIR) is None:
             # todo: double-check source, shouldn't it be Source.ENV?
-            self.update({"root_dir": os.path.abspath(os.getcwd())}, source=Source.BASE)
+            try:
+                self.update(
+                    {"root_dir": os.path.abspath(os.getcwd())}, source=Source.BASE
+                )
+            except FileNotFoundError:
+                wandb.termwarn("Can't get current working directory")
```

Comment:
but what will happen in this case... seems like we need to error out and not just ignore this directory?

Reply:
yea, right. I reverted this change, we should just _error_ out here.